src/www/ui/user-edit.php

```diff
@@ -183,6 +183,8 @@ private function DisplayForm($UserRec, $SessionIsAdmin)
       $SelectedFolderPk = $UserRec['root_folder_fk'];
       $vars['folderListOption'] = FolderListOption($ParentFolder = -1, $Depth = 0, $IncludeTop = 1, $SelectedFolderPk);
     }
+      $SelectedDefaultFolderPk = $UserRec['default_folder_fk'];
+      $vars['folderListOption2'] = FolderListOption($ParentFolder = -1, $Depth = 0, $IncludeTop = 1, $SelectedDefaultFolderPk);
```

Comment:
It will be better to use root folder id as the parent folder for `FolderListOption()` keeping the list sane. Right now, I can select a default folder for a user which is not accessible by them.

Reply:
Fixed as requested!